lib/hyacinth/adapters/digital_object_search_adapter/solr/document_generator.rb

```diff
@@ -23,9 +23,12 @@ def merge_core_fields_for!(digital_object, solr_document = {})
             'digital_object_type_ssi' => digital_object.digital_object_type,
             'doi_ssi' => digital_object.doi,
             'identifier_ssim' => digital_object.identifiers.to_a,
-            'title_ssi' => indexable_title,
+            'title_ssi' => digital_object.generate_title,
```

Comment:
if this isn't separately used as an index, could it just be `title_ss` now?

Reply:
If a user searches the full title of an object (with the non sort portion) would it still show up in the search? Would it only show up in the search if the non sort portion had only stop words in it?

Reply:
it depends on how we want to do the title search, but I am assuming that would be a separate solr field, indexed as text from the configured fields. But it's possible that `digital_object.generate_title` should be passed to `add_keywords` instead of `indexable_title`. Here I was just saying that if we only use the `title_ssi` field to show, it doesn't need to be separately indexed (unless I am missing a use case!)

Reply:
ahh okay that makes sense. So I should convert title_ssi to title_ss and then change the title that is indexed for keyword search. I can make that change.

Reply:
Done.